src/stravalib/protocol.py

```diff
@@ -200,7 +209,12 @@ def exchange_code_for_token(
             "expires_at": response["expires_at"],
         }
         self.access_token = response["access_token"]
-        return access_info
+
+        if return_athlete:
+            athlete_info = SummaryAthlete.model_validate(response["athlete"])
```

Comment:
Since `athlete` is undocumented, any response without an athlete would crash this with a `KeyError`. To prevent that, we could return `response.get("athlete")`. This method would then always return a tuple.

Reply:
I always forget about .get when using dictionaries. Much better call!